### PR TITLE
fix(rabbitmq): control connections closed by broker

### PR DESCRIPTION
**Pull request by franciscorode**

### CODE REVIEW

**Comment by franciscorode** [comment]
I have done a test in staging and it works

**Review by fgsalomon — APPROVED** [review]

File: pytest.ini [hunk 1]
```diff
@@ -8,6 +8,8 @@ env =
     D:MONGODB_USERNAME=root
```
Comment (acostapazo) [inline pytest.ini:8]: should we delete mongodb envvars? I guess so

File: pytest.ini [hunk 2]
```diff
@@ -8,6 +8,8 @@ env =
...
     D:MONGODB_PASSWORD=password123
     D:MONGODB_DATABASE=petisco
+    D:RABBITMQ_MAX_ATTEMPTS_TO_RECONNECT_CONSUMER=3
```
Comment (acostapazo) [inline pytest.ini:11]: I like to have this configuration as a envar. Maybe we should add `PETISCO_` prefix as with another petisco configurations

Comment (acostapazo) [reply pytest.ini:11]: I've just committed an update, I've add a preliminary section in the documentation to gather all the configurations (dummy doc for now)

**Review by acostapazo — APPROVED** [review]

File: pytest.ini (hunk 2 above)
Comment (franciscorode) [reply pytest.ini]: :ok_hand:

**Pull request merged**